java/test/jmri/jmrix/AbstractMonPaneTestBase.java

```diff
@@ -119,6 +119,8 @@ public void testFreezeButton() {
         s.enterTextInEntryField("foo");
         s.clickEnterButton();
         s.clickFreezeButton();
+
+	new org.netbeans.jemmy.QueueTool().waitEmpty(100);
```

Comment:
You have a TAB character instead of spaces on the beginning of the line.

Would it be possible (and easy) to add a test that checks if there is TABs instead of spaces in the beginning of the lines in Java? I have done this misstake several times myself. I would like CI to catch these if possible. There are 1209 files in src folder and 3436 files in test folder with TABs. If you want, I could change these to spaces, if a test is added to CI to check that these doesn't come back.

Reply:
> You have a TAB character instead of spaces on the beginning of the line.

That should be fixed now.

I'll have to fix the editor settings on this machine.
 
> Would it be possible (and easy) to add a test that checks if there is TABs instead of spaces in the beginning of the lines in Java? I have done this misstake several times myself. I would like CI to catch these if possible. There are 1209 files in src folder and 3436 files in test folder with TABs. If you want, I could change these to spaces, if a test is added to CI to check that these doesn't come back.

That's probably something we could do with checkstyle.  @rhwood @bobjacobsen thoughts?

Reply:
A google search gives this:

FileTabCharacterCheck
https://checkstyle.sourceforge.io/apidocs/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheck.html

Reply:
Checkstyle FileTabCharacterCheck works fine.

https://github.com/JMRI/JMRI/compare/master...danielb987:check_tabs

https://travis-ci.com/github/danielb987/JMRI/jobs/323268641

If this is desired, I can fix the java files and create a PR with this.